ykllvmwrap/src/jitmodbuilder.cc

```diff
@@ -608,6 +704,7 @@ class JITModBuilder {
     // module, we must remap them to point to new values in the JIT module.
     llvm::RemapInstruction(NewInst, VMap, RF_NoModuleLevelChanges);
     VMap[&*I] = NewInst;
+    insertAOTMap(I, NewInst, CurBBIdx, CurInstrIdx);
```

Comment:
We talked about using a method to update the VMap and the AOTMap at once in the same place. Is it still possible?

Reply:
At the moment there's only a single place where we need to update the AOTMap. I'm not sure there'll be many other places (probably only constants), as we don't need to map things like function definitions.